middleware_auth_user.go

```diff
@@ -31,13 +31,8 @@ func (umc *UserMiddlewareConfiguration) Validate() error {
 	catcher := grip.NewBasicCatcher()
 
 	if !umc.SkipCookie {
-		if umc.CookieName == "" {
-			catcher.New("must specify cookie name when cookie authentication is enabled")
-		}
-
-		if umc.CookieTTL < time.Second {
-			catcher.New("cookie timeout is less than a second")
-		}
+		catcher.NewWhen(umc.CookieName == "", "must specify cookie name when cookie authentication is enabled")
+		catcher.NewWhen(umc.CookieTTL < time.Second, "cookie timeout is less than a second")
```

Comment:
Can we change this to `must specify a cookie timeout greater than or equal to a second`?